code/src/index.css

```diff
@@ -11,3 +11,148 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/*Entire site styling*/
+.site-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/*Form styling*/
+.form-container {
+  background-color: rgb(243, 242, 242);
+  border: solid 1px black;
+  width: 300px;
+  padding-top: 10px;
+  margin-top: 15px;
+  box-shadow: 10px 10px;
+}
+
+.thought-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-text {
+  font-size: 14px;
+  margin: 10px;
+}
+
+.input-bar {
+  margin: 10px;
+  padding: 10px;
+  width: 250px;
+}
```

Comment:
(Just a suggestion) I would change the input field to text area instead, add a:
- word-break: break-word;
so when you reach the end of the line when writing it breaks and continues writing on the second line.

Reply:
Nice, thanks!